drivers/ninaw10/nina_wifi_bsp.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * This file is part of the OpenMV project, https://openmv.io.
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2013-2021 Ibrahim Abdelkader <iabdalkader@openmv.io>
+ * Copyright (c) 2013-2021 Kwabena W. Agyeman <kwagyeman@openmv.io>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * NINA-W10 driver BSP implementation.
+ */
+#if MICROPY_PY_NETWORK_NINAW10
+#include <stdint.h>
+#include <string.h>
+
+#include "py/runtime.h"
+#include "py/mphal.h"
+#include "modmachine.h"
+#include "extmod/machine_spi.h"
+#include "mpconfigboard.h"
+
+#include "nina_bsp.h"
+#include "nina_wifi_drv.h"
+
+#define debug_printf(...) mp_printf(&mp_plat_print, __VA_ARGS__)
+
+mp_obj_t mp_wifi_spi;
```

Comment:
Can this be static?  Or can one just use `MP_STATE_PORT(mp_wifi_spi)` instead?

Reply:
I was just following this: https://github.com/micropython/micropython/blob/master/ports/rp2/mpbthciport.c#L91